client/src/components/Icon/Icon.tsx

```diff
@@ -13,13 +13,26 @@ const Icon: React.FunctionComponent<IconProps> = ({
   name,
   className,
   title,
-}) => (
-  <>
-    <svg className={`icon icon-${name} ${className || ''}`} aria-hidden="true">
-      <use href={`#icon-${name}`} />
-    </svg>
-    {title && <span className="visuallyhidden">{title}</span>}
-  </>
-);
+}) => {
+  if (name && name.startsWith('M')) {
```

Comment:
If we re-read the docs we should be checking if the `name` is an array, and then adjust the rendering accordingly. 

```suggestion
  if (Array.isArray(name)) {
```

Also, checking if the string starts with a specific character is very risky, any icon may randomly break. E.g. if we ever add a Monkey or Money icon. 🐒🙊

I know most icons are lower case now but just explaining why we would want to avoid this.

Reply:
I agree with this. Thanks. I will make the necessary change.

Reply:
@lb-  I am not sure why after checking that `name` is `array` and then using `name.join()`. It doesn't work.